### PR TITLE
docs: adr for the FE framework / UI library

### PR DESCRIPTION
**Pull request by tylerkron**

### CODE REVIEW

**Review by JacobBrauchler — APPROVED** [review]

File: docs/adr/0003-fe-stack.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+Date: 2023-07-20
+
+## Status
+
+Accepted
+
+## Context
+
+We need to decide what javascript framework to use to make the first version of Fettle-UI. FIDIL is currently a small startup with limited frontend experience. For this reason, it's best to use a framework that we are most experienced. At this stage, boring tech is better than cutting edge tech.
```
Comment (milehiimikey) [inline docs/adr/0003-fe-stack.md:11]: I take it we don't want anyone who likes Vue? We think its boring and not cutting edge, so if you like it.... 🖕

File: docs/adr/0003-fe-stack.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+## Decision
+
+We will use [vue.js](https://vuejs.org/) using [Nuxt](https://nuxt.com/). We will also use [PrimeVue](https://primevue.org/) as it is a popular component library.
```
Comment (milehiimikey) [inline docs/adr/0003-fe-stack.md:15]: This doesn't tell me why its fixing the context of the problem.

File: docs/adr/0003-fe-stack.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+
+## Consequences
+
+Using a popular tech stack that we are more familiar with will allow us to iterate faster than learning new tech stacks from scratch. Using a component library will allow us to make the UI look decent without strong designer skills.
+
+Our FE tech stack will include:
+
+- [Vue.js](https://vuejs.org/)
+- [Nuxt](https://nuxt.com/)
+- [PrimeVue](https://primevue.org/)
```
Comment (milehiimikey) [inline docs/adr/0003-fe-stack.md:25]: This ADR says its tech stack, this seems like its naming 3 of the FE tools used in a much larger tech stack.

File: docs/adr/0003-fe-stack.md [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+
+## Alternatives
+
+There are a bajillion Component Libraries. We might choose a different one after getting experience with the chosen one in this ADR. Alternatives include:
```
Comment (milehiimikey) [inline docs/adr/0003-fe-stack.md:29]: Why didn't we choose any of these?

**Review by milehiimikey — CHANGES_REQUESTED** [review]

File: docs/adr/0003-fe-stack.md (hunk 1 above)
Comment (tylerkron) [reply docs/adr/0003-fe-stack.md:11]: Good point, reworded

File: docs/adr/0003-fe-stack.md (hunk 2 above)
Comment (tylerkron) [reply docs/adr/0003-fe-stack.md:15]: Added more context to this.

File: docs/adr/0003-fe-stack.md (hunk 3 above)
Comment (tylerkron) [reply docs/adr/0003-fe-stack.md:25]: Another good point, renaming to FE Framework

File: docs/adr/0003-fe-stack.md (hunk 4 above)
Comment (tylerkron) [reply docs/adr/0003-fe-stack.md:29]: Left a comment about that.

**Review by milehiimikey — APPROVED** [review]

**Pull request merged**